src/androidTest/java/com/owncloud/android/lib/resources/shares/UpdateShareRemoteOperationIT.kt

```diff
@@ -156,7 +156,10 @@ class UpdateShareRemoteOperationIT : AbstractIT() {
         val capabilityResult = GetCapabilitiesRemoteOperation().execute(nextcloudClient)
         Assert.assertTrue(capabilityResult.isSuccess)
         val capability = capabilityResult.singleData as OCCapability
-        if (capability.version.isNewerOrEqual(NextcloudVersion.nextcloud_21)) {
+
+        if (capability.version.isNewerOrEqual(NextcloudVersion.nextcloud_22)) {
+            assertEquals("Password needs to be at least 10 characters long.", result.message)
+        } else if (capability.version.isNewerOrEqual(NextcloudVersion.nextcloud_21)) {
```

Comment:
can be dropped since it equals the `else` assertion

Reply:
No, this is different, as there is no "." for <21, but "." at the end for 21 and >=22 it is 10 chars.
#confusing

Reply:
right, missed the "."